content/participate/help.adoc

```diff
@@ -12,11 +12,12 @@ tags:
 
 There are many platforms and channels where you can help jenkins users.
 
-==== Mailing Lists, Chats and Interest Groups
+==== Mailing Lists, Chats/Interest Groups, How-To Guides
 
 - https://jenkins.io/mailing-lists[Mailing Lists]
 - https://jenkins.io/chat/[Gitter rooms and IRC channels]
 - https://jenkins.io/sigs/[Special Interest Groups]
+- https://jenkins.io/participate/how-to-guides/[How-To Guides]
```

Comment:
Interesting that the other links on this page are using the full URL.  I thought that the recommendation was to use URL's without specifying the host portion of the URL so that it can be tested locally in development.
```suggestion
- link:/participate/how-to-guides/[How-To Guides]
```

Reply:
You followed the earlier examples in the file perfectly.  Good job.  Unfortunately, they were written in such a way that they make testing a development site more difficult.

Reply:
That's a great idea. Removing actual name of he site will make testing easier.